### ISSUE TITLE
Transparency preview for P3/oklch not working

### ISSUE THREAD

**Issue by megla-tlanghorst** [evilmartians/oklch-picker#124]
Hi, it seems like the transparency preview for oklch isn't working, I think there's something wrong with the formatting of the `oklch()` css values:
![image](https://github.com/evilmartians/oklch-picker/assets/71644972/557d36e2-9c7e-4343-969c-069e8098da25)
Current (wrong) value: `oklch(85.5% 0.357 144.21 / 45)`
Correct value: `oklch(85.5% 0.357 144.21 / 45%)`
With the corrected value:
![image](https://github.com/evilmartians/oklch-picker/assets/71644972/f9d87b78-7252-426f-bff2-7d9820853c27)
Basically, there's just a percent sign missing.

**Comment by ai** [comment]
Thanks for the report.

Fixed 90dadad
